### ISSUE TITLE
Azure cuda driver

### ISSUE THREAD

**Issue by evelinaametova** [CCPPETMR/SIRF-SuperBuild#273]
To make gadgetron and CIL work we had to downgrade CUDA driver.

To uninstall I did following:
```
$ sudo /usr/local/cuda-X.Y/bin/uninstall_cuda_X.Y.pl
$ sudo /usr/bin/nvidia-uninstall
$ sudo apt-get --purge remove nvidia*
$ sudo apt-get --purge remove cuda*
$ sudo rm -r /usr/local/cuda-X
$ sudo reboot
```

to install (note, we need update 1, not 2)
```
$ wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-ubuntu1804.pin
$ sudo mv cuda-ubuntu1804.pin /etc/apt/preferences.d/cuda-repository-pin-600
$ wget https://developer.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda-repo-ubuntu1804-10-1-local-10.1.168-418.67_1.0-1_amd64.deb
$ sudo dpkg -i cuda-repo-ubuntu1804-10-1-local-10.1.168-418.67_1.0-1_amd64.deb
$ sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub
$ sudo apt-get update
$ sudo apt-get install cuda
$ sudo reboot
```

**Comment by paskino** [comment]
on sirf2 I couldn't even run `nvidia-smi` which probes the video card, getting an error of conflicting versions. 

I followed the above instructions and got 
```
Thu Oct 17 11:59:23 2019
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 418.67       Driver Version: 418.67       CUDA Version: 10.1     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  Tesla K80           On   | 00002586:00:00.0 Off |                    0 |
| N/A   35C    P8    33W / 149W |     17MiB / 11441MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0      1493      G   /usr/lib/xorg/Xorg                             9MiB |
|    0      1756      G   /usr/bin/gnome-shell                           6MiB |
+-----------------------------------------------------------------------------+
```

**Comment by KrisThielemans** [comment]
@bathomas I guess this is fixed? Can you please close mentioning the relevant commit?

**Comment by bathomas** [comment]
Fixed by 72e0edd.
